ote_sdk/ote_sdk/algo_backends/test_helpers/__init__.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
Are you sure that extra level of nesting (I mean "algo_backends") is necessary? I'd say that "utils" is a better place for this test suite.

Reply:
These are test helpers for algo backends tests, not for OTE SDK tests, and I would like to make emphasis on it.
Note that there are many OTE SDK tests, and they have their own test helpers -- see them in the file `ote_sdk/ote_sdk/tests/test_helpers.py`.

Reply:
I agree with @druzhkov-paul , `algo_backends` is something unknown for SDK

Reply:
IMHO this is very informative.
It is the only place where "algo_backends" are mentioned in OTE SDK, but I do not know a better name for it.

Just "utils" is not sufficient, since these files are not for utils or for help making tests in OTE SDK -- it help making tests for external projects that use OTE SDK as an interface.

Reply:
After consideration -- fixed: moved `algo_backends/test_helpers -> test_suite`